Lunch/ViewController.swift

```diff
@@ -31,23 +53,35 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
     }
 
     override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
-        
+        if let detailView = segue.destinationViewController as? TodaysLunchViewController {
+            if let eateryIndex = lunchTableView.indexPathForSelectedRow()?.row {
+                detailView.textForLunchLabel = eateries[eateryIndex]
+                detailView.textForVotesLabel = "\(votes[eateryIndex])"
+                detailView.selectedIndex = lunchTableView.indexPathForSelectedRow()!.row as Int
+            }
+
+        }
+
     }
 
     func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return self.items.count
+        return self.eateries.count
     }
 
     func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        var cell: UITableViewCell = self.lunchTableView.dequeueReusableCellWithIdentifier(lunchItemCellIdentifier) as! UITableViewCell
-
-        cell.textLabel?.text = self.items[indexPath.row]
-
-        return cell
+        var cell: UITableViewCell? = self.lunchTableView.dequeueReusableCellWithIdentifier(lunchItemCellIdentifier) as? UITableViewCell
```

Comment:
Lately I've been doing it this way to avoid the force-unwrapping

``` swift
let cell = lunchTableView.dequeueReusableCellWithIdentifier(lunchItemCellIdentifier) as? UITableViewCell ?? UITableViewCell(style: .Subtitle, reuseIdentifier: lunchItemCellIdentifier)
```